packages/e2e/src/specs/overview/overview-in-app-navigation.e2e.ts

```diff
@@ -22,9 +25,137 @@ describe("Overview screen check", () => {
     await about.click()
 
     const sarLink = await OverviewPage.checkSARInformationButton
     await expect(sarLink).toBeDisplayed()
+    await browser.executeAsync((done) => {
+      setTimeout(done, 2000)
+    })
+  })
+
+  it("Check Disconnect button is displayed", async () => {
+    await NavigationTabs.overviewTabClick()
+
+    const disconnectButton = await OverviewPage.disconnectButton
+    await disconnectButton.waitForDisplayed({ timeout: 4000 })
+    await expect(disconnectButton).toBeDisplayed()
+  })
+
+  it("Check battery level and corresponding battery icon", async () => {
+    await NavigationTabs.overviewTabClick()
+
+    const batteryLevel = await OverviewPage.batteryLevel
+    await batteryLevel.waitForDisplayed({ timeout: 4000 })
+    await expect(batteryLevel).toBeDisplayed()
+    const batteryLevelValue = await batteryLevel.getText()
+    const batteryValueFloat = parseFloat(batteryLevelValue)
+
+    if (batteryValueFloat === 100) {
+      const batteryLevelIcon = await OverviewPage.fullBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    } else if (batteryValueFloat > 80) {
+      const batteryLevelIcon = await OverviewPage.veryHighBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    } else if (batteryValueFloat > 60) {
+      const batteryLevelIcon = await OverviewPage.highBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    } else if (batteryValueFloat > 40) {
+      const batteryLevelIcon = await OverviewPage.mediumBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    } else if (batteryValueFloat > 20) {
+      const batteryLevelIcon = await OverviewPage.lowBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    } else if (batteryValueFloat > 0) {
+      const batteryLevelIcon = await OverviewPage.veryLowBatteryIcon
+      await expect(batteryLevelIcon).toBeDisplayed()
+    }
+  })
+
+  it("Check network name and signal strength icon", async () => {
+    const networkName = await OverviewPage.networkName
+    await networkName.waitForDisplayed({ timeout: 4000 })
+    await expect(networkName).toBeDisplayed()
+
+    await expect(networkName).toHaveTextContaining([
+      "Plus",
+      "Orange",
+      "Play",
+      "T-Mobile",
+    ])
+
+    const oneOfThoseShouldBeDisplayed = [
+      await OverviewPage.LowRangeIcon.isDisplayed(),
+      await OverviewPage.MediumRangeIcon.isDisplayed(),
+      await OverviewPage.HighRangeIcon.isDisplayed(),
+      await OverviewPage.VeryHighRangeIcon.isDisplayed(),
+    ]
+
+    expect(oneOfThoseShouldBeDisplayed.filter(Boolean)).toHaveLength(1)
+  })
+
+  it("Check offline mode", async () => {
+    const networkName = await OverviewPage.networkName
+
+    await expect(networkName).not.toBeDisplayed()
+
+    const noRangeIcon = OverviewPage.noRangeIcon
+    await noRangeIcon.waitForDisplayed({ timeout: 4000 })
+    expect(noRangeIcon).toBeDisplayed()
+  })
```

Comment:
Not sure if I get it - what determines that one time the `OverviewPage.networkName` is displayed and the next one is not? The preparation phase of those tests looks the same (nothing is clicked etc.)

Reply:
I had doubts myself.. this test can be run in few different ways: 
- check if network name is displayed and depending on the results 2nd check if there is noRange icon or any other range icons so that would fail only in case network range icon does not match the network name being displayed 
- with split on test preparation I know i would be running no network scenarios hence I disable the other test and by running the only one I am sure network name isn't displayed for my test campaign. by running the two separate tests I know the expected is fail for one of these tests and by looking at the results I know what exactly which case was it was. 
2nd option gives more controll, but perhaps this should be moved to separate file (eg. no network tests)

Reply:
https://github.com/mudita/mudita-center/pull/1058/commits/47caac34271fa41d50a81f8c8d9c7f0141e75069